sensor/common/centralclient/grpc_connection.go

```diff
@@ -81,10 +83,11 @@ func (f *centralConnectionFactoryImpl) SetCentralConnectionWithRetries(conn *uti
 	f.okSignal.Reset()
 	opts := []clientconn.ConnectionOption{clientconn.UseServiceCertToken(true)}
 
-	// waits until central is ready and has a valid license, otherwise it kills sensor by sending a signal
+	// Waits until central is ready and has a valid license, otherwise it kills sensor by sending a signal.
+	// This ping runs over HTTP and does check the health of gRPC connection.
 	if err := f.pingCentral(); err != nil {
-		log.Errorf("checking central status failed: %v", err)
-		f.stopSignal.SignalWithError(errors.Wrap(err, "checking central status failed"))
+		log.Errorf("checking central status over HTTP failed: %v", err)
+		f.stopSignal.SignalWithError(errors.Wrap(err, "checking central status over HTTP failed"))
 		return
 	}
 
```

Comment:
Actually, `SensorComponentEventCentralReachableHTTP` could be emitted already in this line.